### PR TITLE
AVS-485 Send device model details to SyncDevice

### PR DESCRIPTION
**Pull request by You-Hsuan**

### CODE REVIEW

**Review by adrianaudienceview — APPROVED** [review]

File: Core/UIDeviceHelper/UIDevice+ModelDescription.swift [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        case "iPhone12,5":                              return "iPhone 11 Pro Max"
+        case "iPhone12,8":                              return "iPhone SE (2nd generation)"
+        case "iPhone13,1":                              return "iPhone 12 mini"
+        case "iPhone13,2":                              return "iPhone 12"
+        case "iPhone13,3":                              return "iPhone 12 Pro"
+        case "iPhone13,4":                              return "iPhone 12 Pro Max"
+        case "iPhone14,4":                              return "iPhone 13 mini"
+        case "iPhone14,5":                              return "iPhone 13"
+        case "iPhone14,2":                              return "iPhone 13 Pro"
+        case "iPhone14,3":                              return "iPhone 13 Pro Max"
```
Comment (rossp-av) [inline Core/UIDeviceHelper/UIDevice+ModelDescription.swift:52]: Where is the iPhone 14?
Is there a way to achieve this without manually specifying? If not, how will we remember to update every time a new device comes out?

File: Core/UIDeviceHelper/UIDevice+ModelDescription.swift [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+//  UIDevice+ModelDescription.swift
+//  CLArena
+//
+//  Created by Ross Patman on 12/02/2018.
+//  Copyright © 2018 ConcertLive. All rights reserved.
+//
+
+import UIKit
+
+extension UIDevice {
```
Comment (rossp-av) [inline Core/UIDeviceHelper/UIDevice+ModelDescription.swift:11]: Is this code also in Frontier, and does it need to be removed if so?

Comment (You-Hsuan) [reply Core/UIDeviceHelper/UIDevice+ModelDescription.swift:11]: @rossp-av  Yes, I'll remove it from Frontier once I import this SDK version. Will update in the same PR: https://bitbucket.org/livestyled-dev/frontier.ios/pull-requests/2675

File: Core/UIDeviceHelper/UIDevice+ModelDescription.swift (hunk 1 above)
Comment (You-Hsuan) [reply Core/UIDeviceHelper/UIDevice+ModelDescription.swift:52]: @rossp-av  Yes it's difficult for us to remember. That's the same reason for me to ask if it is critical to send it: https://livestyled.atlassian.net/browse/AVS-485?focusedCommentId=102056
Maybe we need to create a Tech improvement ticket in the backlog to review/update device model as well as deployment target every year.

Comment (You-Hsuan) [reply Core/UIDeviceHelper/UIDevice+ModelDescription.swift:52]: @rossp-av sure, will create the ticket. and the iPhone 14 is now added. Thank you!

**Review by rossp-av — APPROVED** [review]

**Pull request merged**